config/resources/domain.json

```diff
@@ -352,10 +352,59 @@
           "predicate": "org:classification",
           "target": "administrative-unit-classification-code",
           "cardinality": "one"
+        },
+        "governing-bodies": {
+          "predicate": "besluit:bestuurt",
+          "target": "governing-body",
+          "cardinality": "many",
+          "inverse": true
         }
       },
       "new-resource-base": "http://data.lblod.info/id/bestuurseenheden/"
     },
+    "governing-bodies": {
+      "name": "governing-body",
+      "class": "besluit:Bestuursorgaan",
+      "attributes": {
+        "name": {
```

Comment:
I copied this from loket, but I don't see it in the data, so maybe not needed here?

Reply:
How do you mean?

Reply:
When I looked at the governing body data in the DB, there was no prefLabel predicate. I just copied this config from Loket, but we might not need it here.

Reply:
But not attached directly to the "besluit:Bestuursorgaan" types, right? At least not for the ones I checked.

Anyway, I leave it as-is, even if we don't use the attribute?